main/models/brapi_pheno_models.py

```diff
@@ -92,7 +92,16 @@ class EntryType(Enum):
     CHECK = 'CHECK'
     TEST = 'TEST'
     FILLER = 'FILLER'
-
+# TODO: We should consolidad and verify that PositionCoordinateType{X,Y} can be generalized
```

Comment:
it can be, the X and Y enums can be deleted

Reply:
Have generalized version now.